packages/inventory-insights/src/Insights.js

```diff
@@ -10,7 +10,7 @@ import { flatten, sortBy } from 'lodash';
 
 import API from './Api';
 import AnsibeTowerIcon from '@patternfly/react-icons/dist/js/icons/ansibeTower-icon';
-import { Battery } from '@redhat-cloud-services/frontend-components/components/Battery';
+import { Battery } from '@redhat-cloud-services/frontend-components/components/esm/Battery';
```

Comment:
I think this should depend in the target env. This will duplicate certain assets in CJS builds